src/filters.hpp

```diff
@@ -23,9 +27,9 @@ class Filters : public Nan::ObjectWrap {
 
     static Nan::Persistent<v8::FunctionTemplate>& constructor();
 
-    void add_filter(filter_key_type&& key, filter_value_type&& filter, zoom_type minzoom, zoom_type maxzoom) {
+    void add_filter(filter_key_type&& key, filter_value_type&& filter, filter_properties properties, zoom_type minzoom, zoom_type maxzoom) {
         // add a new key/value pair, with the value equaling a tuple 'filter_values_type' defined above
-        filters.emplace(key, std::make_tuple(std::move(filter), minzoom, maxzoom));
+        filters.emplace(key, std::make_tuple(std::move(filter), properties, minzoom, maxzoom));
```

Comment:
You'll need `std::move(properties)` here after you use `filter_properties && properties` above.

Reply:
👍 got it(pass the memory into the next target), changed in next submit.